apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/TransformerTestUtils.java

```diff
@@ -1477,8 +1477,9 @@ static void assertEobCommonClaimHeaderData(
     assertEquals(TransformerUtils.buildEobId(claimType, claimId), eob.getIdElement().getIdPart());
 
     if (claimType.equals(ClaimType.PDE))
-      assertHasIdentifier(CcwCodebookVariable.PDE_ID, claimId, eob.getIdentifier());
-    else assertHasIdentifier(CcwCodebookVariable.CLM_ID, claimId, eob.getIdentifier());
+      assertHasIdentifier(CcwCodebookVariable.PDE_ID, String.valueOf(claimId), eob.getIdentifier());
+    else
+      assertHasIdentifier(CcwCodebookVariable.CLM_ID, String.valueOf(claimId), eob.getIdentifier());
```

Comment:
Sidenote: Is there consensus on if/else without brackets in bfd? I don't really care for it. (I could write a checkstyle for this in the future, if we care)

Reply:
yeah. I'm used to having a checkstyle for this too and it can save some errors. lets put it on the tech debt list and fix everywhere at once.

Reply:
fyi, added it.